### ISSUE TITLE
Label "decision document ID" not read with the copy button 

### ISSUE THREAD

**Issue by alisan16** [department-of-veterans-affairs/caseflow#12100]
The label Decision Document ID is not read with the button to copy this number. The label decision Document ID should be read with the number as well as the function that it is a button. (New: 08/22/2019) the lablels need to be read with AT

<img width="1312" alt="Screen Shot 2019-09-16 at 9 41 44 AM" src="https://user-images.githubusercontent.com/1198106/64962910-49ec5680-d866-11e9-8252-1e69ae9b4985.png">
